### PR TITLE
Responsive buttons and view port detection

### PR DESCRIPTION
**Pull request by kaspernj**
Asana: https://app.asana.com/0/7005399303210/144237111800035
Task: https://tasks.kaspernj.org/tasks/2747

### CODE REVIEW

File: app/assets/javascripts/bootstrap_builders/bb-btn-responsive.coffee [hunk 1]
```diff
@@ -0,0 +1,11 @@
+$ ->
+  $.bbUpdateResponseButtons = ->
+    view_port = $.bbViewPort()
```
Comment (guandai) [inline app/assets/javascripts/bootstrap_builders/bb-btn-responsive.coffee]: Where did you use this var ?

File: app/assets/stylesheets/bootstrap_builders/bb-btn-responsive.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.bb-view-port-xs, .bb-view-port-sm {
+  .bb-btn-responsive .bb-btn-label {
+    display: none;
```
Comment (guandai) [inline app/assets/stylesheets/bootstrap_builders/bb-btn-responsive.scss]: Why this not in bb gem ?

File: app/assets/javascripts/bootstrap_builders/view-port-detection.coffee [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    xs: [0, 767],
+    sm: [768, 991],
+    md: [992, 1199],
+    lg: [1200, 999999]
+  }
+
+  $.bbViewPort = ->
+    width = window.innerWidth
+
+    for size, width_and_height of sizes
```
Comment (guandai) [inline app/assets/javascripts/bootstrap_builders/view-port-detection.coffee]: Does width_and_height used and necessary here?  Also in line 23 and line 38

File: app/assets/stylesheets/bootstrap_builders/bb-btn-responsive.scss (hunk 2 above)
Comment (kaspernj) [inline app/assets/stylesheets/bootstrap_builders/bb-btn-responsive.scss]: It is? Check the URL ;-)

File: app/assets/javascripts/bootstrap_builders/bb-btn-responsive.coffee (hunk 1 above)
Comment (kaspernj) [inline app/assets/javascripts/bootstrap_builders/bb-btn-responsive.coffee]: Nice catch! Removed in 2a52969

File: app/assets/javascripts/bootstrap_builders/view-port-detection.coffee (hunk 3 above)
Comment (kaspernj) [inline app/assets/javascripts/bootstrap_builders/view-port-detection.coffee]: @guandai Nope :-) Removed in 48c302a

File: app/assets/stylesheets/bootstrap_builders/bb-btn-responsive.scss (hunk 2 above)
Comment (guandai) [inline app/assets/stylesheets/bootstrap_builders/bb-btn-responsive.scss]: Sorry , my bad,  I thought this is Naoshi project...  :)

**Pull request merged**